misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss

```diff
@@ -9733,4 +9725,4 @@ h3 {
 #chart, #chart_intf_in, #chart_intf_out, #chart_top_ports, #chart_top_sources, #traffic_graph_widget_chart_in, #traffic_graph_widget_chart_out {
   background-color: #FFF;
   border: 1px solid #c2c2c2;
-}
+}
```

Comment:
Please teach your editor to set a trailing newline in the last line of the file. 😊

Reply:
Yes, i will do....thx for mergin. Regards rene